<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="testAudit"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="43.597">
  <properties />
            <testcase name="testNewSignDetail" time="1.707" classname="interfacescript.audit.Audit">
                <failure type="java.lang.IndexOutOfBoundsException"
                  message="Index: 4, Size: 0"
              >
<![CDATA[java.lang.IndexOutOfBoundsException: Index: 4, Size: 0
  java.util.ArrayList.rangeCheck(ArrayList.java:653)
  java.util.ArrayList.get(ArrayList.java:429)
  keeper.appScript.Keeper_Audit.s_newSignDetail(Keeper_Audit.java:82)
  interfacescript.audit.Audit.testNewSignDetail(Audit.java:84)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:74)
  org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:124)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="testLogin" time="37.391" classname="interfacescript.audit.Audit" />
                <testcase name="testNewSignList" time="4.499" classname="interfacescript.audit.Audit" />
      
</testsuite>
